scripts/apps/master-desk/components/FilterBarComponent.tsx

```diff
@@ -1,11 +1,13 @@
 import {gettext} from 'core/utils';
+import _ from 'lodash';
```

Comment:
To reduce bundle size, only import individual functions.

`import {isObject} from 'lodash';`